src/components/AppMap.vue

```diff
@@ -66,6 +66,99 @@
           <code>region</code>: Tower region name (e.g. Tabantha, Lanayru, Hateno) <br>
           <code>fieldarea</code>: Field Map area (0 - 93)<br>
           <code>lotm</code>: Spawns with Lord of the Mountain (1 or 0)<br>
+          <code>location</code>: items possibly in, beneath or above something<br>
+          <span class="ml-3">
+            <span @click="showLocations = !showLocations">
```

Comment:
This should probably be an [HTML5 `<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). Also, `<ul>` inside `<span>` is illegal IIRC.